docs/user/security/rbac_tutorial.asciidoc

```diff
@@ -0,0 +1,119 @@
+[[space-rbac-tutorial]]
+=== Tutorial:  Use Role Based Access Control (RBAC) to customize Kibana Spaces
+
+Role Based Access Control (RBAC) is used to provide access to data, tools,
+and Kibana Spaces.  In this tutorial you will learn how to configure roles
+that provide the right users with the right access to the data, tools, and
+Kibana Spaces.
+
+[float]
+==== Scenario
+
+Our user is a web developer working on a bank's
+online mortgage service.  The web developer has these 
+three requirements:
+
+* Have access to the data for that service 
+* Build visualizations and dashboards
+* Monitor the performance of the system
+
+You'll provide the web developer with the access and privileges to get the job done.
+
+[float]
+==== Prerequisites
+
+To complete this tutorial, you'll need the following:
+
+*  **Administrative privileges**: To create a space, role, and a user.  
+Log in as a superuser (the `elastic` user is a superuser by default).
+*  A **space**: In this tutorial we use `Dev Mortgage` as the space 
+name. See <<spaces-managing, view, create, and delete spaces>> for 
+details on creating a space.
+*  **Data**:  You can use <<tutorial-sample-data, sample data>> or 
+live data.  In the steps below Filebeat and Metricbeat data are used.
+
+[float]
+==== Steps
+
+With the requirements in mind, here are the steps that you will work 
+through in this tutorial:
+
+* Create a role named `mortgage-developer`
+* Give the role the ability to monitor the Elastic Stack
+* Give the role permission to access the data in the relevant indices
+* Give the role permission to create visualizations and dashboards 
+* Create the web developer's user account with the proper roles
+
+[float]
+==== Create a role named `mortgage-developer`
```

Comment:
Not sure `mortgage-developer` is required in this title.

Reply:
agreed.